app/src/main/res/xml/video_audio_settings.xml

```diff
@@ -105,6 +105,15 @@
             android:summary="@string/minimize_on_exit_summary"
             android:title="@string/minimize_on_exit_title"/>
 
+        <ListPreference
+                app:iconSpaceReserved="false"
```

Comment:
Maybe the play icon could be shown here, but I'd prefer there to be an icon.

Reply:
I'm not sure what you wanted to say by this. Do you want me to show play icon? if so, why? There is no any icon inside the preference page and one icon on a long page will look strange